README.md

```diff
@@ -271,6 +271,39 @@ Add to your repository as `.github/workflows/sile.yaml`.
 This work flow assumes your project has a source file `my-document.sil` and will leave behind a `my-document.pdf`.
 Note the comments in [the section about Docker](#docker) regarding version tags.
 
+## Installing third-party packages
+
+Third-party SILE packages rely on the `luarocks` package manager.
+
+To get your Lua version which is used for the execution of `sile`:
+
+```bash
+#!/bin/bash
+# This command finds SILE, then reads its shebang line, and calls whatever the
+# interpreter for SILE is to find its version.
+SILE=$(which sile) || >&2 echo "No sile in '$PATH'?" && \
+  [[ $(head -c2 "$SILE") = '#!' ]] && \
+  LUA=$(head -n1 "$SILE" | cut -c 3-) && \
+  [[ -x $LUA ]] && LUAV=$($LUA -v) && echo $LUAV && \
+  export ___LUA_VERSION=`(awk '{print $2}' | cut -c 1-3) <<< "$LUAV"` && \
+  [[ $___LUA_VERSION =~ ^5\.[1-4]$ ]] || \
+  (>&2 printf 'Cannot auto-detect Lua version!? ' && unset ___LUA_VERSION)
+>&2 printf 'Lua binary: %s\n' "$(which lua$___LUA_VERSION)"
+```
+
+To install the latest development version, for the given package, you may often use its provided “rockspec”:
+
+```bash
+# This example uses markdown.sile, a highly useful plugin that converts Markdown documents to SILE `.sil`.
+luarocks --lua-version $___LUA_VERSION install --server=https://luarocks.org/dev markdown.sile --local
```

Comment:
Two things here (besides the above note about Lua versions). First the extra server argument is only necessary for things that are published to non-default manifests, and the need for that argument should probably be documented in the plugin docs more so than here. Once things are published to the default manifest this shouldn't be necessary. Second we should probably note the various options besides local for project, user, and system wide installations.